ProceduralForest/CMakeLists.txt

```diff
@@ -1,20 +1,31 @@
 cmake_minimum_required(VERSION 3.15)
 project(371ProceduralForest)
 
+
+
 set(CMAKE_CXX_STANDARD 14)
+set(OpenGlLinkers -lglut -lGLU -lGL) # setting all the Glut libraries as one variable.
 
 find_package(GLEW REQUIRED)
-find_package(glfw3 CONFIG REQUIRED)
-find_package(glm CONFIG REQUIRED)
+find_package(glfw3  REQUIRED)
```

Comment:
Was this preventing CMake from finding the libraries? I'm not able to compile the project from your branch it says glew cant be found and using a system shared library leads to shader not compiling

Reply:
missing the OpenGLLinkers for it